pkg/apis/dashboard_manifest.go

```diff
@@ -11,8 +11,6 @@ import (
 	"github.com/grafana/grafana-app-sdk/app"
 )
 
-var ()
```

Comment:
Think this is some kind of merge issue. Will fix when I rebase on main after the conditional rendering PR is merged